docs/monitoring/tag-support.md

```diff
@@ -1,125 +1,113 @@
-title: Monitoring Tags Support
-description: Extract tags from metric sources, environment, assign metadata to host/server/container with custom tags and create more useful dashboards and alerts for your AWS, Docker, Node.js and other applications, hosts and containers
+title: Monitoring Tags 
+description: Tag is an attribute of a data point (or metric) that could be used for grouping and filtering.
 
-## What's a Tag?
+Tags are sent by Sematext Agents as part of every data point and they are shown in UI as filters. 
+For example, as part of Docker container metrics, the agent sends hostname on which the container is running, container identifier, container name, container image as tags which appear as filters in UI. User can then group or filter the container metrics using these tags.
 
-To help you manage your metrics, hosts, and containers, and to help you create more useful dashboards, you can assign metadata to each
-host/server/container in the form of *tags*.
+The tags are automatically collected by agents while fetching metrics and sent as part of metrics to the receiver.
 
-Tags let you organize your hosts/servers/containers in different ways – for example by role, owner, or environment. Each tag consists of
-a name and a value, separated by the ':' character. Both name & value are case-sensitive.
+## Special Tags
 
-We recommend that you devise a set of tag names that meet your needs for each piece of your infrastructure and to keep the tag set small and clean. Using a consistent and not overly broad set of tag names makes it easier for you make the most of Sematext and avoid chaos. Tags will help you to create Alerts for hosts/servers/containers under certain tags or add dashboard widgets based on tags you have defined.
+Following tags are treated specially in Sematext Cloud and cannot be used in Tag aliases, App agent YAMLs and as custom fields in logs. There are used to correlate data across multiple apps and logs. Refer to [Special Logs Fields](/logs/special-fields/) for list of special fields in logs.
```

Comment:
Following tags are treated specially in Sematext Cloud and cannot be used in Tag aliases, App agent YAMLs and as custom fields in logs. There are used to correlate data across multiple apps and logs. Refer to [Special Logs Fields](/logs/special-fields/) for list of special fields in logs.
=>
The following tags are treated as special in Sematext Cloud and cannot be used in tag aliases, app agent YAMLs and as custom fields in logs. There are used to correlate data across multiple apps and logs. Refer to [Special Logs Fields](/logs/special-fields/) for list of special fields in logs.

Reply:
Changed to 
>Following tags are treated as special in Sematext Cloud and cannot be used in tag aliases, App agent YAMLs and as custom fields in logs. There are used to correlate data across multiple Apps. Refer to [Special Logs Fields](/logs/special-fields/) for list of special fields in logs.

Reply:
Can you just make this final edit?
> **The** following tags are treated as special in Sematext Cloud and cannot be used in tag aliases, App agent YAMLs and as custom fields in logs. **They** are used to correlate data across multiple Apps. Refer to Special Logs Fields for list of special fields in logs.